python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -325,25 +325,20 @@ def printSummaryTree(self, level=0, verbosity_level=0, whitespace_marker='', out
         from Ganga.GPIDev.Base.VPrinter import VSummaryPrinter
         self.accept(VSummaryPrinter(level, verbosity_level, whitespace_marker, out, selection, interactive))
 
-    def __eq__(self, _node):
-
-        node = _node
-
+    def __eq__(self, node):
         if self is node:
-            return 1
-        if not node:  # or not self._schema.isEqual(node._schema):
-            return 0
+            return True
```

Comment:
True, but there should not be any `(a == b) is 1` checks. Since all this is doing is changing `__eq__` so that it conforms to the [Python Data Model](https://docs.python.org/2.7/reference/datamodel.html?highlight=__eq__#object.__eq__) ("By convention, `False` and `True` are returned for a successful comparison.") there should be no conflict.

In the future I would like to change the Schema so that it used `True` and `False` rather than `0` and `1` but I agree that it will need some care. It would be nice to finally start using a feature that's been around for 13 years now.